### DISCUSSION TITLE
cp2015 package

### DISCUSSION THREAD

**Discussion by breussf** [community/community#124670] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Dear Paulo Felipe,
tried your github package cp2105 which replicates the Caliendo and Parro (2015) model to estimate the NAFTA effects in R.
Question: is it possible to make changes in the data file cp2015_nafta.rda? One can write parts of this data file into an Excel-File and then change some data. After that one can convert the xlsx file into an rda file.
The question is now how to merge the changed part of the cp2015_nafta file with the old file cp2015_nafta.rda file to implement the changes?
Or is there another possibility to edit (make changes) in the file cp2015_nafta.rda?
Thank you for any hint in advance.
Best regards,
Fritz

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @breussf ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by ebndev** [comment]
Hi @breussf,

Thanks for being a part of the GitHub Community, we're glad you're here!

It looks like you're looking for support from a specific project. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Reply by breussf** [reply]
Thanks for your answer.
As far as my problem is concerned it would be the best to contact the author (Paulo Felipe) who has posted the R-program "paulofelipe/cp2015" on https://github.com/paulofelipe/cp2015

I could not find his email address. Do you have the contact data of Paulo?

Best regards,
Fritz

**Reply by ebndev** [reply]
Hi! 

For the suggestions you had, I would suggest [opening an issue](https://github.com/paulofelipe/cp2015/issues) in the project's repository 😄 

They did not leave contact info in the project's README so I'm not sure the best way to contact them.

**Comment by breussf** [comment]
Hi!

Thank you for your help. In the meantime I have solved my problem.

Best regards,
Fritz


Von: ebndev ***@***.***>
Gesendet: Montag, 3. Juni 2024 18:23
An: community/community ***@***.***>
Cc: Breuss, Fritz ***@***.***>; Mention ***@***.***>
Betreff: Re: [community/community] cp2015 package (Discussion #124670)

External Sender


Hi!

For the suggestions you had, I would suggest opening an issue<https://github.com/paulofelipe/cp2015/issues> in the project's repository 😄

They did not leave contact info in the project's README so I'm not sure the best way to contact them.
